### DISCUSSION TITLE
PHP maybe use Intelephense instead

### DISCUSSION THREAD

**Discussion by kresdjan** [joaotavora/eglot#1136] [Ideas]
It appears that PHP's [php-language-server](https://github.com/felixfbecker/php-language-server) is no longer maintained. Last update was in 2018. So maybe Eglot should be preconfigured to use another package, maybe someting [intelephense](https://intelephense.com/)?

**Comment by nemethf** [comment]
https://github.com/joaotavora/eglot/pull/1105
